edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet/versioning/ModuleManagementHttpClientVersioned.cs

```diff
@@ -23,14 +23,18 @@ abstract class ModuleManagementHttpClientVersioned
         const string LogsUrlTailParameter = "tail";
         const string LogsUrlSinceParameter = "since";
 
+        const int MaxRetryCount = 4;
```

Comment:
why 4?

Reply:
I'm just using whatever the value was there and then +1 it.